Test/TestsDatabase/OrderDatabaseTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Linq;
+using Anlab.Core.Domain;
+using AnlabMvc.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+using Shouldly;
+using Test.Helpers;
+using Xunit;
+
+namespace Test.TestsDatabase
+{
+    [Trait("Category", "DatabaseTests")]
+    public class OrderDatabaseTests
+    {
+        [Fact]
+        public void test1()

```

Comment:
@jSylvestre something like a class that implements iDisposible and opens up a new connection and wraps everything in a try/catch/finally.